### PR TITLE
Law: instantaneous power of force

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/dynamics/instantaneous_power_is_force_times_velocity.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## F - magnitude of force
+## v - speed
+## phi - angle between force and velocity vectors
+
+power = Symbol("power", units.power)
+force = Symbol("force", units.force)
+speed = Symbol("speed", units.speed)
+angle = Symbol("angle", angle_type)
+
+law = Eq(power, force * speed * cos(angle))
```
Comment (blackyblack) [inline symplyphysics/laws/dynamics/instantaneous_power_is_force_times_velocity.py:28]: Derive from https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/laws/dynamics/vector/instantaneous_power_is_force_dot_velocity.py

Comment (alesanter) [reply symplyphysics/laws/dynamics/instantaneous_power_is_force_times_velocity.py:28]: I'm waiting for #324 to do this

Comment (blackyblack) [reply symplyphysics/laws/dynamics/instantaneous_power_is_force_times_velocity.py:28]: Add TODO and we will merge it.

Comment (alesanter) [reply symplyphysics/laws/dynamics/instantaneous_power_is_force_times_velocity.py:28]: I finished the dot product one, by the way

**Review by blackyblack — APPROVED** [review]

**Pull request merged**